lua/swenv/api.lua

```diff
@@ -23,27 +37,45 @@ end
 M.get_venvs = function(venvs_path)
   local success, Path = pcall(require, 'plenary.path')
   if not success then
-    vim.notify('Could not require plenary: '..Path, vim.log.levels.WARN)
+    vim.notify('Could not require plenary: ' .. Path, vim.log.levels.WARN)
     return
   end
   local scan_dir = require('plenary.scandir').scan_dir
 
-  local paths = scan_dir(venvs_path, {depth = 1, only_dirs =true})
   local venvs = {}
+
+  -- CONDA
+  local conda_env_path = Path.parent(Path.parent(Path:new(vim.fn.getenv('CONDA_EXE')))) .. '/envs'
```

Comment:
[nit] I find the following more readable but not super-important:
```suggestion
  local conda_env_path = Path:new(vim.fn.getenv('CONDA_EXE')):parent():parent() .. '/envs'
```